src/StatusQ/Controls/StatusBaseInput.qml

```diff
@@ -51,27 +52,29 @@ Item {
     implicitWidth: 448
     implicitHeight: multiline ? Math.max(edit.implicitHeight + topPadding + bottomPadding, 44) : 44
 
-    Rectangle {
+    Item {
```

Comment:
Actually i dont understand why you create two components instead of one?

Reply:
What do you mean?

Reply:
I think the reason this was done, was so that the `Rectangle` could be made invisible while keeping the element around.

Reply:
In this case isn't it better to use an Item and set its visibility to false instead? Rectangles are expensive when it comes to performance (due to gl calls) since they suppose to paint on screen and it's recommendable to use only when we need to really paint (eg have some background color)

Reply:
I was just trying to interpret what this change from `Rectangle` to `Item` + the introduction of another `Rectangle` below was about :D

Reply:
Probably some clipping issue, removed now as I switched to use a TextInput instead

Reply:
Cool!